### ISSUE TITLE
Slight correction to the target/focus system regarding names

### ISSUE THREAD

**Issue by Shibva** [TeamPneumatic/pnc-repressurized#1027]
### Describe the feature

the feature/suggestion is simple
![image](https://user-images.githubusercontent.com/38770780/173245572-01008cf8-45eb-466a-ae95-25a5e83f623f.png)
when wanting to specify a name it should have to be surrounded in quotes

### Reasons why it should be considered

in command to specify a specific name you would need to surround it in quotes, not to mention the same being needed when using data commands to specify a string 

also, it's a bit confusing and maybe contradicting; what if (and for the sake of the example) you had a pet pig named Creeper? how would the targeting system be able to differentiate between the mob creeper and the entity named creeper? it creates a conflict and while this is a niche case it still is confusing to newcomers like me

**Comment by Shibva** [comment]
Update: turns out it will account for the mob and the name of them. if this was a turret and you had a pet for whatever reason was named "Creeper" it will target both creepers and entities named Creeper

**Comment by desht** [comment]
Yeah, that is an issue with filters, I agree.  Requiring quotes is not a bad idea - that would match only players with that name, or entities with a custom name (e.g. named by Nametag or some other modded mechanism).

**Comment by desht** [comment]
I have been experimenting with this, but my concern right now is that any change could cause an unexpected change in some existing build, which isn't desirable.  I'm thinking this might need to wait until 1.19.
